code/__HELPERS/sorts/helpers.dm

```diff
@@ -0,0 +1,97 @@
+/// Sorts the list in place with timSort, default settings.
+#define SORT_TIM(to_sort, associative) \
+	if (length(to_sort) < 2) { \
+		return to_sort; \
```

Comment:
Dirty return in this macro? Seems like undesired behavior

Reply:
Wdym dirty here 🤔

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#language---hygienic